app/models/docker_builder_service.rb

```diff
@@ -91,6 +119,20 @@ def project
     @build.project
   end
 
+  def registry_credentials
+    return nil unless ENV['DOCKER_REGISTRY'].present?
+    {
+      username: ENV['DOCKER_REGISTRY_USER'],
+      password: ENV['DOCKER_REGISTRY_PASS'],
+      email: ENV['DOCKER_REGISTRY_EMAIL'],
```

Comment:
email is deprecated and will be removed in docker 1.13 IIRC. Maybe we can remove it now?